src/bson_util/mod.rs

```diff
@@ -164,128 +169,30 @@ where
         .ok_or_else(|| D::Error::custom(format!("could not deserialize u64 from {:?}", bson)))
 }
 
-pub fn doc_size_bytes(doc: &Document) -> u64 {
```

Comment:
These were no longer needed as we now just use the length of the serialized vec.